src/main/resources/js/condec.context.menu.js

```diff
@@ -104,57 +104,67 @@
 			var parentId = conDecTreant.findParentId(id);
 			conDecAPI.deleteLink(parentId, id, "i", "s", conDecAPI.setSentenceIrrelevant(id, function() {
 				conDecObservable.notify();
+				JIRA.trigger(JIRA.Events.REFRESH_ISSUE_PAGE, [JIRA.Issue.getIssueId()]);
```

Comment:
That's great! But JIRA.trigger cannot be found on decision knowledge page. 
[issue] How to call JIRA.trigger only for the comment tab panel instead of calling it globally? [/issue]

Reply:
Good point! I might know a solution. Will try it later and report..

Reply:
[decision] Integrate JIRA.trigger into updateView method of ConDec issue module [/decision]